examples/asr/asr_adapters/eval_asr_adapter.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+# Evaluate an adapted model
+
+python eval_asr_adapter.py \
+    --config-path="conf/" \
+    --config-name="asr_adaptation.yaml" \
+    model.pretrained_model=null \
+    model.nemo_model=null \
+    model.adapter.adapter_name=<name of the adapter to evaluate> \
+    model.test_ds.manifest_filepath="<Path to validation/test manifest>" \
+    model.test_ds.batch_size=16 \
+    model.train_ds.manifest_filepath=null \
+    model.validation_ds.manifest_filepath=null \
+    model.adapter.in_features=null \
+    trainer.devices=1 \
+    trainer.precision=32
+
+# Pretrained Models
+
+For documentation on existing pretrained models, please visit -
+https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/asr/results.html
+
+"""
+
+import pytorch_lightning as pl
+from omegaconf import OmegaConf, open_dict
+
+from nemo.collections.asr.models import ASRModel
+from nemo.core import adapter_mixins
+from nemo.core.config import hydra_runner
+from nemo.utils import logging
+from nemo.utils.exp_manager import exp_manager
+
+
+def update_encoder_config_to_support_adapter(model_cfg):
```

Comment:
update_encoder_config_to_support_adapter and update_model_cfg looks like to be duplicated in both train and evaluation scripts. Can we move them into a single place?

Reply:
Maybe in the future, right now the first draft of this script modifies just encoder, we expect more advanced adapters in different places so I don't want to move this inside nemo yet.